cluster/src/main/java/org/apache/iotdb/cluster/server/ClientServer.java

```diff
@@ -90,6 +91,12 @@
    */
   private MetaGroupMember metaGroupMember;
```

Comment:
As you have abstract coordinator to cooperate with metaGroupMember and dataGroupMember, I think maybe it's better to remove metaGroupMember in ClientServer to decouple clientServer(service logic) and metaGroupMember(internal logic).

Reply:
ok